jina/types/arrays/search_ops.py

```diff
@@ -112,3 +113,24 @@ def shuffle(self, seed: Optional[int] = None) -> 'DocumentArray':
         from .document import DocumentArray
 
         return DocumentArray(self.sample(len(self), seed=seed))
+
+    def split(self, attribute: str) -> Dict[Any, 'DocumentArray']:
```

Comment:
If it is only implemented with splitting by tags, then this arg should be called as `tag`

```suggestion
    def split(self, tag: str) -> Dict[Any, 'DocumentArray']:
```

also in docstring